daemon/execdriver/lxc/driver.go

```diff
@@ -85,16 +85,21 @@ func (d *driver) Run(c *execdriver.Command, pipes *execdriver.Pipes, startCallba
 		dataPath = d.containerDir(c.ID)
 	)
 
+	container, err := d.createContainer(c)
```

Comment:
c.ProcessConfig.Termal is true or not doesn't accect createContainer.
So, I think if we can confirm error occurs in createContainer, we can avoid running codes dealing with c.ProcessConfig